src/cmd/create.go

```diff
@@ -634,10 +635,9 @@ func pullImage(image, release string) (bool, error) {
 
 	logrus.Debugf("Pulling image %s", imageFull)
 
-	if logLevel := logrus.GetLevel(); logLevel < logrus.DebugLevel {
+	if logLevel := logrus.GetLevel(); logLevel < logrus.DebugLevel && isatty.IsTerminal(1) {
 		s := spinner.New(spinner.CharSets[9], 500*time.Millisecond)
 		s.Prefix = fmt.Sprintf("Pulling %s: ", imageFull)
-		s.Writer = os.Stdout
```

Comment:
Is there a reason to remove this line?

Reply:
After a few rounds of testing it seems accidental to me, so I filed https://github.com/containers/toolbox/pull/498 to put it back.

But let me know if I am missing something. :)